blogs/cli/index.md

```diff
@@ -0,0 +1,444 @@
+```json
+{
+  "AgentIP": "10.0.1.76",
+  "Bytes": 561,
+  "DnsErrno": 0,
+  "Dscp": 20,
+  "DstAddr": "f904:ece9:ba63:6ac7:8018:1e5:7130:0",
+  "DstMac": "0A:58:0A:80:00:37",
+  "DstPort": 9999,
+  "Duplicate": false,
+  "Etype": 2048,
+  "Flags": 16,
+  "FlowDirection": 0,
+  "IfDirection": 0,
+  "Interface": "ens5",
+  "K8S_FlowLayer": "infra",
+  "Packets": 1,
+  "Proto": 6,
+  "SrcAddr": "3e06:6c10:6440:2:a80:37:b756:270f",
+  "SrcMac": "0A:58:0A:80:00:01",
+  "SrcPort": 46934,
+  "TimeFlowEndMs": 1709741962111,
+  "TimeFlowRttNs": 121000,
+  "TimeFlowStartMs": 1709741962111,
+  "TimeReceived": 1709741964
+}
+```
+- `./output/flow/<CAPTURE_DATE_TIME>.db` database that can be inspected using `sqlite3` for example: 
+```bash
+bash-5.1$ sqlite3 ./output/flow/<CAPTURE_DATE_TIME>.db 
+SQLite version 3.34.1 2021-01-20 14:10:07
+Enter ".help" for usage hints.
+sqlite> SELECT DnsLatencyMs, DnsFlagsResponseCode, DnsId, DstAddr, DstPort, Interface, Proto, SrcAddr, SrcPort, Bytes, Packets FROM flow WHERE DnsLatencyMs >10 LIMIT 10;
+12|NoError|58747|10.128.0.63|57856||17|172.30.0.10|53|284|1
+11|NoError|20486|10.128.0.52|56575||17|169.254.169.254|53|225|1
+11|NoError|59544|10.128.0.103|51089||17|172.30.0.10|53|307|1
+13|NoError|32519|10.128.0.52|55241||17|169.254.169.254|53|254|1
+12|NoError|32519|10.0.0.3|55241||17|169.254.169.254|53|254|1
+15|NoError|57673|10.128.0.19|59051||17|172.30.0.10|53|313|1
+13|NoError|35652|10.0.0.3|46532||17|169.254.169.254|53|183|1
+32|NoError|37326|10.0.0.3|52718||17|169.254.169.254|53|169|1
+14|NoError|14530|10.0.0.3|58203||17|169.254.169.254|53|246|1
+15|NoError|40548|10.0.0.3|45933||17|169.254.169.254|53|174|1
+```
+or `dbeaver`:
+![dbeaver](./images/dbeaver.png)
+
+- `packets` to get pcap file containing [full packet data](https://github.com/netobserv/netobserv-ebpf-agent/blob/main/examples/packetcapture-dump/README.md). 
+
+![packets](./images/packet_table.png)
+
+It will display a table view with latest packets collected and write data under output/pcap directory.
+To stop capturing press `CTRL + C`.
+
+This will write pcap into a single file located in `./output/pcap/<CAPTURE_DATE_TIME>.pcap` that can be opened with Wireshark for example:
+
+![wireshark](./images/wireshark.png)
+
+This command must have both protocol and port specified to work. We will improve the packet capture filtering options in the future.
+Packet drop, DNS latencies and RTT features are not available in this command.
+
+A third option called `cleanup` allows you to remove any remaining netobserv component. That should not happen since this command is automatically run when exiting the capture but it may be necessary to manually execute it in case of connectivity issue or crash.
+
+### Use cases
+
+Let's move on some concrete scenarios. For each of the following use cases, you must have `netobserv CLI` installed and you need to be connected to your cluster using:
+```
+oc login <CLUSTER_API_URL> --username <CLUSTER_ADMIN> --password <PASSWORD>
```

Comment:
they can use any other valid method, such as `oc login --token=... --server=...`, this would also work ; Maybe just don't mention the logging method?